after-sign-hook.js

```diff
@@ -0,0 +1,37 @@
+// See: https://medium.com/@TwitterArchiveEraser/notarize-electron-apps-7a5f988406db
+
+'use strict';
+const fs = require('fs');
+const path = require('path');
+const electronNotarize = require('electron-notarize');
+
+module.exports = async function (params) {
+  if (process.platform !== 'darwin') {
+    return;
+  }
+
+  console.log('afterSign hook triggered', params);
+
+  // Same appId in electron-builder.
+  const appId = 'com.wulkano.kap';
+
+  const appPath = path.join(params.appOutDir, `${params.packager.appInfo.productFilename}.app`);
+  if (!fs.existsSync(appPath)) {
+    throw new Error(`Cannot find app at: ${appPath}`);
+  }
```

Comment:
I don't see the point of this.`electronNotarize` already throws if it can't find the app. Just remove the below try/catch instead.